### ISSUE TITLE
Additional sort options

### ISSUE THREAD

**Issue by Empereol** [MLoughry/sort-typescript-imports#2]
I like having the ability to sort via path but this ends up putting my third party imports at the bottom which I would prefer at the top.

Example of what currently happens with `"typescript.extension.sortImports.sortMethod": "path"` 
```typescript
import { SpinnerService } from "../shared/spinner/spinner.service";
import { Toast } from "../shared/toasts/models/toast";
import { ToastsService } from "../shared/toasts/toasts.service";
import { StorageService } from "./storage.service";
import "rxjs/add/observable/throw";
import "rxjs/add/operator/catch";
import "rxjs/add/operator/map";
import { Observable } from "rxjs/Observable";
import { Http, RequestMethod, Headers } from "@angular/http";
import { Inject, Injectable } from "@angular/core";
```

whereas I think it would be preferable to have an option to flip this to look like
```typescript
import { Inject, Injectable } from "@angular/core";
import { Http, RequestMethod, Headers } from "@angular/http";
import { Observable } from "rxjs/Observable";
import "rxjs/add/observable/throw";
import "rxjs/add/operator/catch";
import "rxjs/add/operator/map";
import { SpinnerService } from "../shared/spinner/spinner.service";
import { Toast } from "../shared/toasts/models/toast";
import { ToastsService } from "../shared/toasts/toasts.service";
import { StorageService } from "./storage.service";
```

**Comment by Empereol** [comment]
Thanks! Though there seems to be a bug with packages that start with a special characters (i.e. `@angular`) are always sorted to the bottom of the imports.

With the `typescript.extension.sortImports.pathSortOrder` set to 
```json
"typescript.extension.sortImports.pathSortOrder": [
    "package",
    "relativeUpLevel",
    "relativeDownLevel"
]
```
my imports are ordered like:
```typescript
import "rxjs/add/observable/throw";
import "rxjs/add/operator/catch";
import "rxjs/add/operator/map";
import { Observable } from "rxjs/Observable";
import { APP_CONFIG } from "../app-config";
import { IAppConfig } from "../app-config.interface";
import { SpinnerService } from "../shared/spinner/spinner.service";
import { Toast } from "../shared/toasts/models/toast";
import { ToastsService } from "../shared/toasts/toasts.service";
import { StorageService } from "./storage.service";
import { Http, RequestMethod, Headers } from "@angular/http";
import { Inject, Injectable } from "@angular/core";
```

**Comment by MLoughry** [comment]
Opened and addressed this issue in #5. I'll be pushing the release shortly.
